app/_layout.tsx

```diff
@@ -1,3 +1,4 @@
+import Providers from '@src/providers';
 import { SplashScreen, Stack } from 'expo-router';
 import { useEffect } from 'react';
 import useFontsLoader from 'src/hooks/use-fonts-loader';
```

Comment:
@MuhammedKpln can we update the imports here to use @src and also update the ts config base url and remove the package.json file in src